pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -72,21 +72,30 @@ type ConsoleServerCLIConfigBuilder struct {
 	authType                   string
 }
 
-func (b *ConsoleServerCLIConfigBuilder) OAuthClientConfig(authConfig *configv1.Authentication, oauthClientSecret *corev1.Secret) *ConsoleServerCLIConfigBuilder {
+func (b *ConsoleServerCLIConfigBuilder) OAuthClientConfig(authConfig *configv1.Authentication) *ConsoleServerCLIConfigBuilder {
 	switch authConfig.Spec.Type {
 	case "", configv1.AuthenticationTypeIntegratedOAuth:
 		b.oauthClientID = api.OAuthClientName
-	case "OIDC", configv1.AuthenticationTypeNone:
-		if oauthClientSecret == nil {
+	case configv1.AuthenticationTypeOIDC, configv1.AuthenticationTypeNone:
+		if len(authConfig.Spec.OIDCProviders) != 1 {
```

Comment:
I think you may want to use `== 0` in these comparisons to make it a bit more future-proof.